rllab/misc/logger.py

```diff
@@ -40,8 +45,8 @@
 _log_tabular_only = False
 _header_printed = False
 
-_tensorboard_default_step = 0
 _tensorboard_step_key = None
+_tensorboard = Summary()
```

Comment:
This seems to be a conflict with `_tensorboard = TensorBoardOutput()` in line 51. Is it possible to merge class Summary with TensorBoardOutput?